src/state/index.js

```diff
@@ -350,6 +356,7 @@ AFRAME.registerState({
     },
 
     leaderboardqualify: state => {
+      if (!state.has6DOFVR) { return; }
```

Comment:
Would there be an equivalent 3DOF leaderboard, if they are disqualified from current one?

Reply:
Yeah not eligible!